controllers/akodeploymentconfig/phases/phases_unit_test.go

```diff
@@ -4,17 +4,21 @@
 package phases
 
 import (
+	"github.com/go-logr/logr"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	akoov1alpha1 "github.com/vmware-samples/load-balancer-operator-for-kubernetes/api/v1alpha1"
+	ako_operator "github.com/vmware-samples/load-balancer-operator-for-kubernetes/pkg/ako-operator"
```

Comment:
suggest moving self packages `github.com/vmware-samples/load-balancer-operator-for-kubernetes/*` to a new section, seperated by the blank line